Decompression/GZipcompressor.cpp

```diff
@@ -0,0 +1,59 @@
+
+#include "GZipCompressor.h"
+#include <sstream>
+
+GZipCompressor::GZipCompressor()
+{
+}
+
+CMR OB;
```

Comment:
It looks like some strange global. What is your rationale to introduce something like this?
See: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#i2-avoid-global-variables